modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -2183,7 +2186,12 @@ private SnapshotView snapshotViewSupplier(SnapshotMetadata meta) {
         List<File> dirs = snapshotCacheDirectories(meta.snapshotName(), null, meta.folderName(), name -> true);
         Collection<String> cacheGrps = F.viewReadOnly(dirs, FilePageStoreManager::cacheGroupName);
 
-        return new SnapshotView(meta.snapshotName(), meta.consistentId(), F.concat(meta.baselineNodes(), ","), F.concat(cacheGrps, ","));
+        return new SnapshotView(
```

Comment:
Can we make constructor with only `meta, cacheGrps` and perform transformation inside view.

Reply:
You also, must regenerate `SnapshotViewWalker` - just run `SystemViewRowAttributeWalkerGenerator` and commit generated changes.